llama_index/indices/managed/google/generativeai/base.py

```diff
@@ -163,12 +163,31 @@ def as_retriever(self, **kwargs: Any) -> BaseRetriever:
         return self._index.as_retriever(**kwargs)
 
     def as_query_engine(
-        self, *, answer_style: int = 1, **kwargs: Any
+        self,
+        *,
+        temperature: float = 0.7,
+        answer_style: Any = 1,
```

Comment:
mnit: can the default use AnswerStyle.ABSTRACTIVE or is there some import funkiness we're trying to avoid?